applications/rag/README.md

```diff
@@ -36,6 +36,17 @@ gcloud container clusters create-auto ${CLUSTER_NAME:?} \
   --release-channel rapid
 ```
 
+### Enable [Private Service Connect](https://cloud.google.com/vpc/docs/private-service-connect)
+
+Note: Skip this step when `create_network = true` is set.
```

Comment:
I guess this is a little confusing since we don't surface create_network = true at the RAG level. Maybe we move this sentence to the /infrastructure README? And the RAG README always suggests checking that PSC is enabled for the default VPC (since that's what we use).

Reply:
Good idea. I moved this note to the `cloudsql` module.